src/main/app/src/app/zaken/intake-afronden-dialog/intake-afronden-dialog.component.less

```diff
@@ -35,6 +38,25 @@ button {
   }
 }
 
+.mat-expansion-panel-header,
+.mat-expansion-panel-content {
+  padding-left: 16px;
+  padding-right: 16px;
+}
+
+.mat-expansion-panel,
+.mat-expansion-panel-header {
+  color: var(--mdc-filled-text-field-label-text-color);
+  font-family: var(--mdc-filled-text-field-label-text-font);
+  font-size: var(--mdc-filled-text-field-label-text-size);
+  font-weight: var(--mdc-filled-text-field-label-text-weight);
+  letter-spacing: var(--mdc-filled-text-field-label-text-tracking);
+}
+
 .suffix {
   float: right;
 }
+
+.mat-mdc-dialog-actions {
+  padding: var(--mat-dialog-with-actions-content-padding, 20px 24px);
+}
```

Comment:
We really need to make a story for theming and streamline this throughout the platform instead of setting this on a per-component basis

Reply:
totally agree; we should have defined expansion styling designed and in place.

but this was looking so bad that, since I was on the CSS anyways, had to align a bit

Reply:
Hey, at least I am taking the theme vars! that is already a step forward 😜